content/rc/security/access-control/data-access-control/configure-acls.md

```diff
@@ -10,6 +10,16 @@ categories: []
 aliases: 
 ---
 
+You can define custom Redis ACL rules to assign to a data access role or use predefined Redis ACLs.
+
+Redis provides three predefined ACL rules, which are marked with the Redis logo:
+
+    - **Full Access**: Allows all commands. 
+    - **Read-Write**: Allows read and write commands and excludes dangerous commands.
+    - **Read-only**: Allows read commands only.
+
+ACLs that are not marked with the Redis logo are user-defined ACL rules.
```

Comment:
Would go with the positive sentence of ACLs that are marked instead of not

Reply:
@dvir-redis I'm not sure what you mean here. If I say "ACLs that are marked with the Redis logo are user-defined ACL rules", that isn't correct - the rules that are marked with the Redis logo are the provided ACL rules.

Reply:
Yes, I meant:
`ACLs that are marked with the Redis logo are Redis-defined ACL rules, and can not be changed.`

Reply:
I'll modify the previous sentence and say that Redis-defined ACL rules can not be changed, but I do want to specifically mention that any rules not marked with the logo are user-defined.